ca/server.go

```diff
@@ -808,3 +826,162 @@ func isFinalState(status api.IssuanceStatus) bool {
 
 	return false
 }
+
+func (s *Server) finishRootRotation(expectedRootCA *api.RootCA) error {
```

Comment:
Random cosmetic thought: If we make this take a `store.Tx` argument, it would remove a level of indentation. The call to `finishRootRotation` could be merged into the `store.Batch` above it.